### PR TITLE
Introduce a configuration that points a public endpoint for the search-host

### PR DESCRIPTION
**Pull request by danquah**
The search-host is accessed both by the admin-backend, and directly by the
administrative-users browser. For situations where the public url for
the search-host is not accessible to the backend (eg. due to firewalls) we need
two seperate configurations for the public url (for the users browser) and the
internal url (for backend interaction).

**Notice**: even tough additions to parameters.yml.dist should lead to the parameters.yml being updated automatically at next deploy, this does not seem to happen.
In other words, prior to deploying this change to an existing environment you have to
introduce the parameter manually into parameters.yml. (I might also just be wrong, but
this has been my experience so far).

### CODE REVIEW

**Comment by tuj** [comment]
Closed since project is no longer maintained
